.gitignore

```diff
@@ -0,0 +1,7 @@
+.bundle
+Gemfile.lock
```

Comment:
Nothing worse that trying to cut a release and you're faced with a failure (on CI but not locally) due to version differences.

![screen shot 2015-08-04 at 08 43 58](https://cloud.githubusercontent.com/assets/44164/9054767/fb6f0c38-3a84-11e5-92c5-908910391832.png)

IMO, you should be able to checkout an old version at any time and the tests should still work and pass.